### ISSUE TITLE
Invalid Escape Sequence

### ISSUE THREAD

**Issue by Gerry2k5** [Gerry2k5/randword#1]
As of the date of writing, this package is now throwing the following error:
```
/usr/local/bin/randword:236: SyntaxWarning: invalid escape sequence '\s'
  if not re.match("^[PS]FX\s", line):
```

For some reason, the \s in the Regex is now flagging as invalid.
The code works as intended, but I am investigating whether a change has been made to the Regex syntax in Python

**Comment by Gerry2k5** [comment]
As per https://docs.python.org/3/library/re.html this was always invalid, just not flagged.
Has been fixed by making the Regex a raw string as per the doc

**Comment by Gerry2k5** [comment]
Fixed in Commit 8a58cf0
